jest.config.js

```diff
@@ -27,9 +27,7 @@ module.exports = {
 	// coverageDirectory: undefined,
 
 	// An array of regexp pattern strings used to skip coverage collection
-	// coveragePathIgnorePatterns: [
-	//   "/node_modules/"
-	// ],
+	coveragePathIgnorePatterns: ['src/third-party-tags/external-scripts'],
```

Comment:
maybe all `external-scripts` folders should be ignored?
```suggestion
	coveragePathIgnorePatterns: ['external-scripts'],
```

Reply:
actually, i wonder if we should move them something like `/vendor` (next to `src`)? it kind of feels wrong that they live in our code?

Reply:
similar to https://github.com/guardian/frontend/tree/main/static

Reply:
I was about to say the same thing!

Reply:
agree that's better.
I pushed the changes for it but there's an error in build step
```
Error: src/third-party-tags/lotame.ts(1,30): error TS6059: File '/home/runner/work/commercial-core/commercial-core/vendor/lotame-script.ts' is not under 'rootDir' '/home/runner/work/commercial-core/commercial-core/src'. 'rootDir' is expected to contain all source files.
Error: src/third-party-tags/twitter-uwt.ts(1,48): error TS6059: File '/home/runner/work/commercial-core/commercial-core/vendor/twitter-script.ts' is not under 'rootDir' '/home/runner/work/commercial-core/commercial-core/src'. 'rootDir' is expected to contain all source files.
```
From reading this is fixable by using rootDirs compiler option 

https://www.typescriptlang.org/docs/handbook/module-resolution.html#virtual-directories-with-rootdirs

Reply:
should be fixed now